docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -0,0 +1,64 @@
+= Installation
```

Comment:
To keep the doc more readable I'd move the logic of creating the download path to the top of the doc and extract it to a variable:

```suggestion
ifeval::["{page-component-display-version}" == "main"]
:releases-download-path: https://github.com/kubearchive/kubearchive/releases/latest/download
endif::[]
ifeval::["{page-component-display-version}" != "main"]
:releases-download-path: https://github.com/kubearchive/kubearchive/releases/download/{page-component-display-version}
endif::[]

= Installation
```

Reply:
Ah, very nice. Done.